tests/apollo/test_skvbc_ro_replica.py

```diff
@@ -103,15 +109,18 @@ class SkvbcReadOnlyReplicaTest(unittest.TestCase):
         bft_network.start_replica(ro_replica_id)
         # TODO replace the below function with the library function:
         # await tracker.skvbc.tracked_fill_and_wait_for_checkpoint(initial_nodes=bft_network.all_replicas(), checkpoint_num=1)     
-        with trio.fail_after(60):  # seconds
+        with trio.fail_after(seconds=60):
             async with trio.open_nursery() as nursery:
                 nursery.start_soon(tracker.send_indefinite_tracked_ops)
                 while True:
-                    with trio.move_on_after(.5):  # seconds
-                        key = ['replica', 'Gauges', 'lastExecutedSeqNum']
-                        lastExecutedSeqNum = await bft_network.metrics.get(ro_replica_id, *key)
-                        # success!
-                        if lastExecutedSeqNum >= 150:
-                            print("Replica" + str(ro_replica_id) + " : lastExecutedSeqNum:" + str(lastExecutedSeqNum))
-                            nursery.cancel_scope.cancel()          
-                
+                    with trio.move_on_after(seconds=.5):
+                        try:
+                            key = ['replica', 'Gauges', 'lastExecutedSeqNum']
+                            lastExecutedSeqNum = await bft_network.metrics.get(ro_replica_id, *key)
+                        except KeyError:
+                            continue
+                        else:
+                            # success!
+                            if lastExecutedSeqNum >= 150:
+                                print("Replica" + str(ro_replica_id) + " : lastExecutedSeqNum:" + str(lastExecutedSeqNum))
+                                nursery.cancel_scope.cancel()
```

Comment:
those are changes I have made into master - why do we see them here?
Please rebase properly before we can merge this PR...